CMakeLists.txt

```diff
@@ -430,16 +430,6 @@ if (SSG_PRODUCT_VSEL)
     add_subdirectory("products/vsel" "vsel")
 endif()
 
-# ZIP only contains source datastreams and kickstarts, people who
-# want sources to build from should get the tarball instead.
-if(SSG_TARGET_OVAL_VERSION VERSION_EQUAL "5.11")
-    ssg_build_zipfile("scap-security-guide-${SSG_VERSION}")
-    ssg_build_vendor_zipfile("scap-security-guide-${SSG_VERSION}-SCAP-1.3")
-else()
-    ssg_build_zipfile("scap-security-guide-${SSG_VERSION}-oval-${SSG_TARGET_OVAL_VERSION}")
-    ssg_build_vendor_zipfile("scap-security-guide-${SSG_VERSION}-SCAP-1.2")
```

Comment:
You probably want to remove just the call of the ssg_build_vendor_zipfile macro, the ssg_build_zipfile needs to stay there, because now ehn you removed it you break the genration of a zipfile. Try to go to the build directory and run `ninja zipfile` there:
```
[jcerny@thinkpad build{pr/9152}]$ ninja zipfile
ninja: error: 'zipfile/scap-security-guide-0.1.63.zip', needed by 'CMakeFiles/zipfile', missing and no known rule to make it
```